elfi/examples/gauss.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import scipy.stats as ss
+import elfi
+from functools import partial
+
+
+"""An example implementation of a Gaussian noise model.
+
+The design of the example is identical to ma2.py
+"""
+
+
+def GNM(mean, var, n_obs=100, batch_size=1, random_state=None):
+    # Standardising the parameter's format.
+    mean = np.asanyarray(mean).reshape((-1, 1))
+    var = np.asanyarray(var).reshape((-1, 1))
+
+    y = ss.norm.rvs(loc=mean, scale=var, size=(batch_size, n_obs),
+        random_state=random_state)
+    return y
+
+
+def ss_mean(x):
+    """The summary statistic corresponding to the mean.
+    """
+    ss = np.mean(x, axis=1)
+    return ss
+
+
+def ss_var(x):
+    """The summary statistic corresponding to the variance.
+    """
+    ss = np.var(x, axis=1)
+    return ss
+
+
+def get_model(n_obs=100, true_params=None, seed_obs=None):
+    """Returns a complete Gaussian noise model
+
+    Parameters
+    ----------
+    n_obs : int
+        the number of observations
+    true_params : list
+        true_params[0] corresponds to the mean,
+        true_params[1] corresponds to the variance
+    seed_obs : None, int
+        seed for the observed data generation
+
+    Returns
+    -------
+    m : elfi.ElfiModel
+    """
+
+    if true_params is None:
+        true_params = [10, 2]
+
+    y_obs = GNM(*true_params, n_obs=n_obs,
+        random_state=np.random.RandomState(seed_obs))
+    sim_fn = partial(GNM, n_obs=n_obs)
+
+    m = elfi.ElfiModel(set_current=False)
+    elfi.Prior('uniform', true_params[0] - 50, true_params[0] + 50,
+        model=m, name='mean')
+    elfi.Prior('uniform', 0, true_params[1] + 5, model=m, name='var')
```

Comment:
I think this would be fine for an example model, as he is just ensuring the priors cover the true value. I'd like to have some other prior than uniform since our other examples already use uniform priors. Nice to have this model though.